### PR TITLE
Temporary: Fix vcpkg PR notification logic

### PR DESCRIPTION
**Pull request by danieljurek**
A change to our codeowners logic caused an unforeseen regression. While we're waiting on more robust logic this PR resolves the GitHub alias from the user who queued the build in DevOps and then adds a comment to the vcpkg PR for that user. 

Example build pipeline run that adds comment: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3144289&view=logs&j=db30d5e9-5c21-5d7b-155a-3df6a2f30198&t=04e70e6c-0739-5054-b0b2-e98b1a777538

PR opened with comment from the person who ran the pipeline: https://github.com/microsoft/vcpkg/pull/34315

### CODE REVIEW

File: eng/pipelines/templates/stages/archetype-cpp-release.yml [hunk 1]
```diff
@@ -193,15 +193,20 @@ stages:
...
                       - pwsh: |
-                          $codeOwnersToNotify = $(Build.SourcesDirectory)/eng/common/scripts/get-codeowners.ps1 `
-                            -TargetDirectory "/sdk/${{ parameters.ServiceDirectory }}/" `
-                            -CodeownersFileLocation "$(Build.SourcesDirectory)/.github/CODEOWNERS"
+                          $notifyUsers = ./eng/scripts/Get-TriggeringUserGitHubAlias.ps1 `
+                            -EmailAddress '$(Build.RequestedForEmail)' `
+                            -ClientId '$(opensource-aad-app-id)' `
+                            -ClientSecret '$(opensource-aad-secret)' `
+                            -TenantId '$(opensource-aad-tenant-id)' `
+                            -Fallback '@rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman'
```
Comment (antkmsft) [inline eng/pipelines/templates/stages/archetype-cpp-release.yml:205]:
```suggestion
                            # cspell:disable
                            -Fallback '@rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman'
                            # cspell:enable
```

**Review by antkmsft — APPROVED** [review]

**Review by ahsonkhan — COMMENTED** [review]
Had a comment about fetching email addresses, otherwise looks good.

**Comment by antkmsft** [comment]
Today Eventhubs have released, and this step was in red. Let's address the feedback if necessary and get this merged. There is going to be one more release this week, for Identity, and ideally there should be no failed steps.

**Comment by danieljurek** [comment]
This is fixed. 

Logs now show only the environment variable -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3160920&view=logs&j=db30d5e9-5c21-5d7b-155a-3df6a2f30198&t=04e70e6c-0739-5054-b0b2-e98b1a777538

Release PR with comment -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3160920&view=logs&j=db30d5e9-5c21-5d7b-155a-3df6a2f30198&t=04e70e6c-0739-5054-b0b2-e98b1a777538

**Comment by ahsonkhan** [comment]
> Release PR with comment

This one?
https://github.com/microsoft/vcpkg/pull/34454

**Pull request merged**